pkg/cache/task.go

```diff
@@ -265,6 +265,12 @@ func (task *Task) handleSubmitTaskEvent(event *fsm.Event) {
 
 	events.GetRecorder().Eventf(task.pod, v1.EventTypeNormal, "Scheduling",
 		"%s is queued and waiting for allocation", task.alias)
+	// if this task is belong to the gang member, then post a message to indicate the pod will being scheduled as gang member
```

Comment:
Suggest changing the text to the following:

```
// if this task belongs to a task group, that means the app has gang scheduling enabled
// in this case, post an event to indicate the task is being gang scheduled
```

Reply:
OK,Sure